tests/tck/features/match/PipeAndVariable.feature

```diff
@@ -79,26 +79,108 @@ Feature: Pipe or use variable to store the lookup results
       | true | 42  |
 
   Scenario: mixed usage of cypher and ngql
+    When executing query:
+      """
+      YIELD ['Tim Duncan', 'Tony Parker'] AS a
+      | UNWIND $-.a AS b
+      | GO FROM $-.b OVER like YIELD edge AS e
+      """
+    Then the result should be, in any order:
+      | e                                                            |
+      | [:like "Tim Duncan"->"Manu Ginobili" @0 {likeness: 95}]      |
+      | [:like "Tim Duncan"->"Tony Parker" @0 {likeness: 95}]        |
+      | [:like "Tony Parker"->"LaMarcus Aldridge" @0 {likeness: 90}] |
+      | [:like "Tony Parker"->"Manu Ginobili" @0 {likeness: 95}]     |
+      | [:like "Tony Parker"->"Tim Duncan" @0 {likeness: 95}]        |
+    When executing query:
+      """
+      YIELD {a:1, b:['Tim Duncan', 'Tony Parker'], c:'Tim Duncan'} AS a
+      | YIELD $-.a.b AS b
+      | UNWIND $-.b AS c
+      | GO FROM $-.c OVER like YIELD edge AS e
+      """
+    Then the result should be, in any order:
+      | e                                                            |
+      | [:like "Tim Duncan"->"Manu Ginobili" @0 {likeness: 95}]      |
+      | [:like "Tim Duncan"->"Tony Parker" @0 {likeness: 95}]        |
+      | [:like "Tony Parker"->"LaMarcus Aldridge" @0 {likeness: 90}] |
+      | [:like "Tony Parker"->"Manu Ginobili" @0 {likeness: 95}]     |
+      | [:like "Tony Parker"->"Tim Duncan" @0 {likeness: 95}]        |
+    When executing query:
+      """
+      YIELD {a:1, b:['Tim Duncan', 'Tony Parker'], c:'Tim Duncan'} AS a
+      | YIELD $-.a.c AS b
+      | UNWIND $-.b AS c
+      | GO FROM $-.c OVER like YIELD edge AS e
+      """
+    Then the result should be, in any order:
+      | e                                                       |
+      | [:like "Tim Duncan"->"Manu Ginobili" @0 {likeness: 95}] |
+      | [:like "Tim Duncan"->"Tony Parker" @0 {likeness: 95}]   |
     When executing query:
       """
       LOOKUP ON player
       WHERE player.name == 'Tim Duncan'
       YIELD player.age as age, id(vertex) as vid
-      | UNWIND $-.vid as a RETURN a
+      | UNWIND $-.vid as a | YIELD $-.a AS a
```

Comment:
Will the column `age` be passed on?

Reply:
will not be passed to the next statement in nGql